etc/omero.properties

```diff
@@ -57,6 +57,11 @@ omero.db.authority=export.openmicroscopy.org
 # will be used by OMERO. Your database installation
 # will need to be configured to accept *at least* as
 # many, preferably more, connections as this value.
+#
+# The values of omero.threads.max_threads and
+# omero.threads.background_threads play a role
+# in how many DB connections are used at any given
+# time.
```

Comment:
Maybe advise to set higher than `omero.threads.background_threads` and default to such?

Reply:
Better, @mtbc?

Reply:
Absolutely. :+1: